ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/provision/cucushift-installer-rehearse-vsphere-upi-provision-chain.yaml

```diff
@@ -12,8 +12,10 @@ chain:
   - ref: upi-conf-vsphere-dns
   - ref: upi-conf-vsphere-ova
   - ref: upi-conf-vsphere-ova-vcm
+  - ref: ipi-conf-vsphere-folder
```

Comment:
Shall we call `ipi-deprovision-vsphere-folder` in its deprovision chain ?

Reply:
IPI will delete folder but UPI will not, added upi-deprovision-vsphere-folder step

Reply:
I think `ipi-deprovision-vsphere-folder` can be shared in both IPI and UPI install workflow, right ?